test/libsolidity/smtCheckerTests/types/array_static_mapping_aliasing_1.sol

```diff
@@ -5,7 +5,6 @@ contract C
 	mapping (uint => uint8)[2] severalMaps8;
 	mapping (uint => uint)[2][2] severalMaps3d;
 	function f(mapping (uint => uint) storage map) internal {
-		// Accesses are safe but oob is reported because of aliasing.
```

Comment:
It seems the problem with aliasing of maps had been fixed somewhere along the way, because no oob was reported here anymore.